modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/UninitializedOperationMessageProcessor.java

```diff
@@ -0,0 +1,266 @@
+    this.resolvedParameters = resolvedParameters;
+  }
+
+  @Override
+  protected void doInitialise() throws InitialisationException {
+    //Do nothing
+  }
+
+  @Override
+  public ReturnDelegate createReturnDelegate() {
+    return operationMessageProcessorDelegate.createReturnDelegate();
+  }
+
+  @Override
+  public TargetReturnDelegate getTargetReturnDelegate() {
+    return operationMessageProcessorDelegate.getTargetReturnDelegate();
+  }
+
+  @Override
+  public ValueReturnDelegate getValueReturnDelegate() {
+    return operationMessageProcessorDelegate.getValueReturnDelegate();
+  }
+
+  @Override
+  public boolean isTargetPresent() {
+    return operationMessageProcessorDelegate.isTargetPresent();
+  }
+
+  @Override
+  public Optional<String> getTarget() {
+    return operationMessageProcessorDelegate.getTarget();
+  }
+
+  @Override
+  public void doStart() throws MuleException {
+    operationMessageProcessorDelegate.doStart();
+  }
+
+  @Override
+  public void doStop() throws MuleException {
+    operationMessageProcessorDelegate.doStop();
+  }
+
+  @Override
+  public void doDispose() {
+    operationMessageProcessorDelegate.doDispose();
+  }
+
+  @Override
+  public ExecutionMediator createExecutionMediator() {
+    return operationMessageProcessorDelegate.createExecutionMediator();
+  }
+
+  @Override
+  public ParameterValueResolver getParameterValueResolver() {
+    return operationMessageProcessorDelegate.getParameterValueResolver();
+  }
+
+  @Override
+  public void resolveParameters(CoreEvent.Builder eventBuilder,
+                                BiConsumer<Map<String, Supplier<Object>>, ExecutionContext> afterConfigurer)
+      throws MuleException {
+    operationMessageProcessorDelegate.resolveParameters(eventBuilder, afterConfigurer);
+  }
+
+  @Override
+  public void disposeResolvedParameters(ExecutionContext<OperationModel> executionContext) {
+    operationMessageProcessorDelegate.disposeResolvedParameters(executionContext);
+  }
+
+
+  @Override
+  protected Map<String, Object> getResolutionResult(CoreEvent event, Optional<ConfigurationInstance> configuration)
+      throws MuleException {
+
+    ValueResolvingContext valueResolvingContext = ValueResolvingContext.builder(event).build();
+    Object config = resolverSet.resolve(valueResolvingContext).asMap().get("config-ref");
+
+    Map resolvedParametersMap = resolvedParameters.entrySet().stream()
+        .filter(e -> e.getKey().equals("config-ref"))
+        .collect(Collectors.toMap(key -> key, value -> {
+          try {
+            return value.getValue().resolve(valueResolvingContext);
+          } catch (MuleException e) {
+            e.printStackTrace();
+          }
+          return null;
+        }));
+
+    return (Map) ImmutableMap.builder().putAll(resolvedParametersMap).put("config-ref", config).build();
+  }
+
+  @Override
+  public void setMuleContext(MuleContext context) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "MuleContext"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7719&issues=AWk7fTH-otIXnmsUoDr8&open=AWk7fTH-otIXnmsUoDr8)

[ISSUE_KEY]: AWk7fTH-otIXnmsUoDr8